docs/bugs.json

```diff
@@ -6,7 +6,7 @@
         "introduced": "0.4.17",
         "fixed": "0.5.0",
         "severity": "very low",
-        "check": {"json-path": {"query": "$..[?(@.name === 'EventDefinition')]..[?(@.name === 'UserDefinedTypeName' && @.attributes.type.startsWith('struct'))]", "astType": "ast-json"}}
+        "check": {"json-path": {"query": "$..[?(@.nodeType === 'EventDefinition')]..[?(@.nodeType === 'UserDefinedTypeName' && @.typeDescriptions.typeString.startsWith('struct'))]", "astType": "ast-compact-json"}}
```

Comment:
One final remark: Would it be better to change it to something like `check: {ast-compact-json: "$..."}`? Otherwise, if we even add multiple checks, it is not clear what the `"astType": "ast-compact-json"` refers to.

Reply:
Yea, I guess that makes sense. It makes it simpler at least.